pubspec.yaml

```diff
@@ -5,6 +5,13 @@ description: OpenStreetMap Nominatim API.
 homepage: https://github.com/enyo/nominatim
 documentation: http://enyo.github.com/nominatim/nominatim.html
 dependencies:
-  xml: '>=0.9.5 <0.9.6'
+  xml: ^3.2.3
+  flutter:
+    sdk: flutter
```

Comment:
I don't think that adding the flutter dependency is necessary here. This will restrict this package for flutter.
Please remove it, and run `pub get` again to update the `pubspec.lock`.

(The same goes for the `dev_dependencies`)